src/services/IssueService.ts

```diff
@@ -138,7 +138,7 @@ function toIssue(source: any, votes: Array<Vote>): Issue {
     id: source.id,
     number: source.number,
     title: source.title,
-    description: source.body,
+    body: source.body,
```

Comment:
Not sure why this was renamed? It doesn't seem to be reflected on any other files/components